theme/src/components/nav-items.js

```diff
@@ -31,10 +31,10 @@ function NavItem({href, children}) {
 function NavItems({items}) {
   return (
     <>
-      <VisuallyHidden>
-        <h3 id="nav-heading">Site navigation</h3>
-      </VisuallyHidden>
-      <NavList aria-labelledby="nav-heading">
+      <NavList aria-label="Site">
+        <VisuallyHidden>
+          <h3 id="nav-heading">Site navigation</h3>
+        </VisuallyHidden>
```

Comment:
Is the `id=nav-heading` still necessary?

Just curious, is there a reason we need `aria-label` and a visually hidden heading?

Reply:
Good question! The heading allows screen reader users to navigate to the footer via heading -- often sr users will navigate through headings to understand page content and structure of the page. The reason we need `aria-label` on the nav instead of `aria-labelledby` the heading is so that the screen reader does not read navigation twice:

i.e. `Site Navigation Navigation`


AND we do want `Site Navigation` as the heading text because if a user navigates via heading `Site` alone would be confusing.

Reply:
Makes sense. Thanks for the explanation!